test/specs/commonTests/isConformant.tsx

```diff
@@ -31,19 +31,25 @@ export default (Component, options: any = {}) => {
   const { throwError } = helpers('isConformant', Component)
 
   const componentType = typeof Component
+  const wrappedComponent = mount(<Component {...requiredProps} />)
```

Comment:
with this change we are breaking tests isolation to some degree - now it could be the case that some state of the component that was set as a matter of previous test will affect next one. Just something that we should be aware of before making this change - we should carefully weight props and cons of this move, and to get idea about pros we would need to measure numbers first (and understand the source of these numbers - in either case, obtain evidence that the thing we are trying to optimize is indeed a source of bottleneck)

Reply:
Totally aware of this, when we really need for some test the component with the exact props provided rendered, we should use the mount, but if we are ok with reusing the existing with just adding some props, this should be enough. This was considered for the tests where it is changed, because there is no real need for the component to be mounted over and over again.

Reply:
yes, but here is the problem that could happen: suppose that now we are totally aware that this particular test is safe to reuse component, because during its execution nothing in terms of component's state is changed/used. Then, on the next day (after the test is introduced) the logic of component changes, and it results in our test checking for wrong condition - because of this isolation invariant (that is, actually, a fundamental assumption) being not satisfied. This is the main factor for me to make it as a deliberate step and only if we are absolutely sure that it is critically important for us